usr/libexec/playtron/hardware-test-tool

```diff
@@ -391,6 +392,87 @@ class DisplayTest(PlaceholderTest):
             self.done = True
 
 
+class SpeakersTest(PlaceholderTest):
+    def __init__(self):
+        self.name = 'Speakers|扬声器'
+        self.reset()
+        self.start_time = None
+        self.time_count = 0
+        self.wav_file_name = '/tmp/test_record.wav'
+        self.msg_headphones_hint = 'Unplug headphones|拔下耳机'
+        self.audio_output_started = False
+        self.audio_input_started = False
+        self.cleanup_started = False
+
+    def start(self, screen, font):
+        self.reset()
+        self.start_time = time.perf_counter()
+
+    def audio_output(self,):
+        # 5.7 seconds is the exact amount of time for the "Front right. Front left." voice line to happen twice.
+        os.popen('timeout 5.7s speaker-test -c 2 -t wav')
+
+    def audio_input_record(self):
+        # Record from the microphone for 6 seconds.
+        subprocess.Popen('arecord -f cd -d 6 ' + self.wav_file_name, shell=True)
+
+    def audio_input_playback(self):
+        os.popen('aplay ' + self.wav_file_name)
+
+    def cleanup(self):
+        try:
```

Comment:
Once the test is complete it just stays at a blank screen which made me think something went wrong. We can set `self.done = True` here to go straight to the Pass/Fail prompt.